.github/workflows/maven.yml

```diff
@@ -14,29 +14,24 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        maven: [3.3.1, 3.5.4, 3.6.3, 3.8.7, 3.9.0]
+        maven: [3.3.1, 3.5.4, 3.6.3, 3.8.7, 3.9.0, 4.0.0-alpha-4]
+        jdk: ["11"]
```

Comment:
Great, I've been thinking that we should add Java versions to the matrix and start cross-testing. Thanks for setting this up. I can add additional versions in a separate change once this is merged.